pt-BR/upgrading_ruby_on_rails.md

```diff
@@ -1,33 +1,36 @@
 **DO NOT READ THIS FILE ON GITHUB, GUIDES ARE PUBLISHED ON https://guides.rubyonrails.org.**
 
-Upgrading Ruby on Rails
+Atualizando Ruby on Rails
 =======================
 
-This guide provides steps to be followed when you upgrade your applications to a newer version of Ruby on Rails. These steps are also available in individual release guides.
+Este guia fornece os passos a serem seguidos quando você for atualizar suas aplicações
+para uma versão mais nova do Ruby on Rails. Estes passos também estão disponíveis em guias de releases individuais.
```

Comment:
```suggestion
para uma versão mais nova do Ruby on Rails. Estes passos também estão disponíveis em guias de lançamento de versão individuais.
```

To procurando algo para traduzir, pensei em colocar algo assim. O que acha?

Reply:
@HenriqueMorato neste caso mantive `releases`  por ser um termo bastante usado e conhecido no meio. Prefere que mude para a tradução? Ou então posso marca-lo como estrangeirismo com itálico.

O que acham?

Reply:
Acho legal sim! Ainda temos que criar um padrão de tradução mas acho ótimo do itálico.